joomla/joomla-cms
# Using MailTemplate to create new Template record #39480

Pull Request for Issue [#39480.](https://github.com/joomla/joomla-cms/issues/39480)

### Summary of Changes
Add extension parameter in createTemplate static function.So that a new record will insert into database after creating email template.


### Testing Instructions
Just make same change as in this PR and create a mail-template and see mail-template after saving in database.


### Actual result BEFORE applying this Pull Request
After creating mail-template a new record not inserted into database in mail template table



### Expected result AFTER applying this Pull Request
After this changes on creating mail-template a new record inserted into database in main template table.


### Link to documentations
Please select:
- [ok] No documentation changes for docs.joomla.org needed

- [ok] No documentation changes for manual.joomla.org needed
